src/scripts/main.js

```diff
@@ -1,3 +1,190 @@
 'use strict';
 
-// write your code here
+const field = document.querySelector('.game-field').firstElementChild;
+const cellList = document.querySelectorAll('TD');
+const startButton = document.querySelector('.start');
+const score = document.querySelector('.game-score');
+const rows = [...field.rows].reduce((res, el, index) => {
```

Comment:
try to use a more descriptive name than 'el'
in this case you can use 'row'
please check other places